src/components/MiroshnikGK/MiroshnikGKElement.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Color from './Color';
+import WithMoveValidation from './integrations/WithMoveValidation';
```

Comment:
```suggestion
import Chess from './integrations/WithMoveValidation';
```
Мне кажется, так будет понятнее